examples/replica/outputs.tf

```diff
@@ -9,7 +9,7 @@ output "id" {
 
 output "replica_id" {
   description = "Postgresql read-only replica instance id"
-  value       = module.read_only_replica_postgresql_db.id
+  value       = module.read_only_replica_postgresql_db[0].id
```

Comment:
Is there any value, even in an example of showing how to output 1 of n ids?

Perhaps this should be replica_ids returning `module.read_only_replica_postgresql_db[*].id`

Reply:
Added this.